package.json

```diff
@@ -4,9 +4,10 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
-    "dev": "vite",
+    "dev": "vite --host",
     "build": "tsc && vite build",
-    "preview": "vite preview"
+    "preview": "vite preview --host",
+    "watch": "tsc && vite preview --host & && vite build --watch"
```

Comment:
```suggestion
    "watch": "tsc && vite preview --host && vite build --watch"
```

Reply:
Not a typo. 😬 
It's so that the preview runs in the background and we can execute the next command.

Reply:
oh okay, it was causing this error on my end 😓

Reply:
```
tamarazuk@Tamaras-MacBook-Pro app-template % npm run watch

> codesandbox-template-vite@0.0.0 watch
> tsc && vite preview --host & && vite build --watch

sh: -c: line 0: syntax error near unexpected token `&&'
sh: -c: line 0: `tsc && vite preview --host & && vite build --watch'
```

Reply:
Removing the parenthesis was the "typo". :grimacing

Reply:
It worked when I tested it, but it was probably cached. 🤦‍♀️